docs/topics/vs-code-extension-interface.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * docs/vsc-extension-guide/master.adoc
+
+
+[id="vs-code-extension-interface_{context}"]
+= {ProductShortName} extension interface
+
+The following components are displayed in the interface of the {ProductName} ({ProductShortName}) extension:
+
+* The Run configuration pane is to the right of the Activity bar. The Run configuration pane displays the applications and files analyzed by MTA and any issues uncovered by the analysis. The Run configuration pane includes a folder icon named *Report*, which you can click to open the MTA Report in your browser.
+
+* The Configuration settings pane, which is in the upper right pane of the extension interface, is where you configure your analyses. When an analysis begins, the lower portion of this pane is covered by the Run configuration detail pane.
```

Comment:
Make sure that this pane is only used for configuration settings. If the pane is used for different things, try to give it a more generic name.

"detail pane" or "details pane"? I prefer details. But I'm not sure that the last sentence is relevant.

Reply:
Spelling of "Analyses"